### PR TITLE
Add mixpanel tracking for type of frontend and contract

### PR DESCRIPTION
**Pull request by icerove**
Add mixpanel tracking for type of frontend and contract user choose

### CODE REVIEW

File: tracking.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+const MIXPANEL_TOKEN = 'df164f13212cbb0dfdae991da60e87f2'
+
+const chalk = require('chalk')
+const mixpanel = require('mixpanel').init(MIXPANEL_TOKEN);
+
+const track = async (frontendType, contractType) => {
+    try {
+        console.log(chalk`
+NEAR collect the type of UI and contract to improve developer experience. But NEAR will never send private information.
```
Comment (volovyk-s) [inline tracking.js:9]: I'm not sure about the grammar here. @TiffanyGYJ can you please take a look?

File: index.js [hunk 2]
```diff
@@ -165,6 +166,8 @@ We suggest that you begin by typing:`)
     {bold ${runCommand} dev}`)
   }
 
+  mixpanel.track(frontend, contract)
```
Comment (volovyk-s) [inline index.js:169]: @icerove I would not put `NEAR collect the type of UI and contract...` info at the end of the build. We really want developers to concentrate on `cd appName` and `yarn dev` commands. I would rather put it at the beginning of the function.

**Review by volovyk-s — CHANGES_REQUESTED** [review]
@icerove Looks good overall.

File: tracking.js (hunk 1 above)
Comment (TiffanyGYJ) [reply tracking.js:9]: I would write something like this: 

NEAR collects basic information on the type of commands used to improve developer experiences. All data are anonymous, therefore, no personal information that could identify you will be shared.

**Comment by TiffanyGYJ** [comment]
I would write something like this:

NEAR collects basic information on the type of commands used to improve developer experiences. All data are anonymous, therefore, no personal information that could identify you will be shared.

**Review by volovyk-s — APPROVED** [review]

**Pull request merged**